src/main/java/com/manardenza/dao/AbstractDao.java

```diff
@@ -0,0 +1,38 @@
+package com.manardenza.dao;
+
+import com.manardenza.entity.AbstractObject;
+import com.manardenza.entity.Hotel;
+import com.manardenza.entity.User;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public abstract class AbstractDao<T extends AbstractObject> {
+
+    protected List<Hotel> hotelList = new ArrayList<>();
+    protected List<User> userlList = new ArrayList<>();
+
+    public T save(T obj) {
+        return obj;
+    }
+
+    public boolean delete(T obj) {
+        return false;
+    }
+
+    public boolean saveAll(List<T> obj) {
+        return false;
+    }
+
+    public List<T> getAll() {
+        return null;
+    }
+
+    public ArrayList<Hotel> getHotelsByName(String hotelName) {
+        return null;
+    }
+
+    public ArrayList<Hotel> getHotelsByCity(String hotelCity) {
+        return null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Return an empty collection instead of null. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1168)